src/ubloxGPS.h

```diff
@@ -643,8 +649,8 @@ class ubloxGPS
     uint32_t getTime(void);
     uint32_t getUTCTime();
     uint8_t  getSatellites(void);
-    int8_t   getGeoIdHeight(void);
-    uint8_t  getHDOP(void);
+    double   getGeoIdHeight(void);
```

Comment:
What are the dimensional units for these methods (meters?) ? Would be good to add either to the method signature or to header doc.  Even better would be a dimensional type (something like meters_t or length_t, where the typedef is double)

Reply:
Meters.  Added some doxygen comments